src/core/qgis.h

```diff
@@ -437,13 +437,12 @@ typedef unsigned long long qgssize;
 #endif
 
 #ifndef QGISEXTERN
-#ifdef Q_OS_WIN
+#ifdef _MSC_VER
```

Comment:
Theuns - I'm unsure about this change. Was this leftover from earlier testing? Didn't we establish that __declspec( dllexport ) was correct for the mingw build?

Reply:
Nyall - With the original code from the repo it used the `__GNUC__` define with the unix style deceleration and the plugins worked regarding loading data, I will have to make sure on Monday, thats why we changed the code like this.
You are right in the sense that `__declspec( dllexport )` is working correctly for all export functions.
So in this case `__declspec( dllexport )`  should work.

Reply:
For reference - off list discussion determined that this change was not necessary